lib/dangermattic/plugins/common/reporter.rb

```diff
@@ -25,8 +25,6 @@ class Reporter < Plugin
     #
     # @return [void]
     def report(message:, type: :warning)
-      return if message.nil? || message.empty? || type.nil?
-
```

Comment:
Thank you for following up on this.

If the behavior changes on Danger's side somehow, or if making sure the behavior on our hand is robust becomes more important, we can always add some tests around what Danger itself does and run them as part of our unit tests to make sure it's consistent with our assumptions. 

But that sounds like overkill at this point in time.